### DISCUSSION TITLE
What is the recommended way to handle asynchronous asyncpg connection in the `teardown_after_execution` of a resource?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18159] [Q&A]
Hi, there. Is there a recommended way to handle an asynchronous asyncpg connection in the `teardown_after_execution` of a resource?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1700490570354809?thread_ts=1700490570.354809&cid=C01U954MEER).*

**Comment by cristiangreco94** [comment]
any update?
